module-check/src/test/clojure/clojure/core/typed/test/core_test.clj

```diff
@@ -0,0 +1,8 @@
+(ns test.core-test
+ (:use clojure.test)
+ (:require [clojure.core.typed :as t])
+ (:require test.core) 
+)
```

Comment:
`:use` is not recommended please use `(:require [clojure.test :refer :all])` instead.
Group all require together:
```clojure
(:require [clojure.core.typed :as t]
           test.core
           ...)
```
And use 2 space indentation so it aligned with clojure styleguide.